lib/Parse/ParseStmt.cpp

```diff
@@ -441,7 +450,7 @@ StmtResult Parser::ParseSEHTryBlock() {
     return StmtError(Diag(Tok, diag::err_expected) << tok::l_brace);
 
   StmtResult TryBlock(ParseCompoundStatement(/*isStmtExpr=*/false,
-                      Scope::DeclScope | Scope::SEHTryScope));
+                      (unsigned)(Scope::DeclScope | Scope::SEHTryScope)));
```

Comment:
As I added additional argument into ParseCompoundStatement method to pass checked scope flag, it is necessary to distinguish from those two methods (ParseCompoundStatement(bool, bool) and ParseCompoundStatement(bool, unsigned).
So, for compilation purpose, I have added above casting

ParseCompoundStatement(bool isStmtExpr) & ParseCompoundSatatment(bool isStmtExpr, unsigned flag)